agent/functional_tests/testdata/taskdefinitions/granular-stop-timeout/task-definition.json

```diff
@@ -0,0 +1,41 @@
+{
+  "family": "ecsftest-container-granular-stop-timeout",
+  "containerDefinitions": [
+    {
+      "name": "parent",
+      "image": "127.0.0.1:51670/busybox:latest",
+      "cpu": 0,
+      "memory": 64,
+      "essential": true,
+      "command": ["sh", "-c", "sleep 15"],
+      "dependsOn": [
+        {
+          "containerName": "dependency1",
+          "condition": "START"
+        },
+        {
+          "containerName": "dependency2",
+          "condition": "START"
+        }
+      ]
+    },
+    {
+      "name": "dependency1",
+      "image": "127.0.0.1:51670/busybox:latest",
+      "cpu": 0,
+      "memory": 64,
+      "essential": false,
+      "command": ["sh", "-c", "trap 'echo caught' SIGTERM; sleep 60"],
+      "stopTimeout": 5
+    },
+    {
+      "name": "dependency2",
+      "image": "127.0.0.1:51670/busybox:latest",
+      "cpu": 0,
+      "memory": 64,
+      "essential": false,
+      "command": ["sh", "-c", "trap 'echo caught' SIGTERM; sleep 60"],
```

Comment:
is there any damage in doing 20 and 30 seconds? Instead of 50 and 60? Just to make test go a bit faster